code/modules/mob/living/silicon/silicon.dm

```diff
@@ -366,7 +366,7 @@
 
 /mob/living/silicon/attack_alien(mob/living/carbon/alien/humanoid/M)
 	if(..()) //if harm or disarm intent
-		var/damage = rand(10, 20)
+		var/damage = 25

```

Comment:
This is a huge buff.

If you're wanting to eliminate randomness, this should be ~15.